content/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md

```diff
@@ -0,0 +1,299 @@
+---
+Title: Redis Enterprise Software release notes 7.4.2-tba (January 2024)
+linkTitle: 7.4.2-tba (January 2024)
+description: New Cluster Manager UI enhancements, including Active-Active database management. TLS 1.3 support. Automatic recovery configuration. IPv6 support for internal traffic. Maintenance mode enhancements. Two feature sets.
+compatibleOSSVersion: Redis 7.2.0
+weight: 72
+alwaysopen: false
+categories: ["RS"]
+aliases: 
+---
+
+​[​Redis Enterprise Software version 7.4.2](https://redis.com/redis-enterprise-software/download-center/software/) is now available!
+
+## Highlights
+
+This version offers:
+
+- New Cluster Manager UI enhancements, including Active-Active database management
+
+- TLS 1.3 support
+
+- Automatic recovery configuration
+
+- IPv6 support for internal traffic
+
+- Maintenance mode enhancements
+
+- Two feature sets
+
+## New in this release
+
+### New features
+
+- Added support for TLS 1.3 and removed support for deprecated TLS versions, 1.0 and 1.1, and deprecated 3DES and RC4 encryption ciphers.
```

Comment:
@rrelledge this phrasing suggests that we didn't have TLS 1.3 support at all until now, which is wrong and misleading. 
(It's my bad since I suggested the current phrasing...)
I believe that we should change this to something like: 

- Added full support for TLS 1.3: 
  - TLS 1.3 cipher suites can now be configured, where applicable (see https://docs.redis.com/staging/DOC-2445/rs/security/encryption/tls/ciphers/)
  - TLS 1.3 can be set as the minimum TLS version (see https://docs.redis.com/staging/DOC-2445/rs/security/encryption/tls/tls-protocols/)

-

Reply:
Fixed